### PR TITLE
Update docs and preset params.

### PR DESCRIPTION
**Pull request by asenina**

### CODE REVIEW

File: docs/compression_algorithms/Quantization.md [hunk 1]
```diff
@@ -305,6 +305,7 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
 ```
 {
     "algorithm": "quantization",
+    "preset": "mixed", // The preset defines the quantization schema for weights and activations. The parameter takes values 'permormance' or 'mixed'. The mode 'performance' defines symmetric weights and activations. The mode 'mixed' defines symmetric 'weights' and asymmetric 'activations'. Weights quantize with 'per_channel': True by default. Any preset parameters can be overriden by sections 'weights' and 'activations'.
```
Comment (AlexKoff88) [inline docs/compression_algorithms/Quantization.md:308]: That is the default preset? I suggest using "performance" and specify it in the document. I mean replace: ` "preset": "mixed"` with ` "preset": "performance"`

Comment (AlexKoff88) [reply docs/compression_algorithms/Quantization.md:308]: `overriden` -> `overridden`. I remember we decided to throw an exception if both preset and manual settings are provided.

Comment (asenina) [reply docs/compression_algorithms/Quantization.md:308]: @AlexKoff88 After additional discussion with @alexsu52 , we decided to make a different behavior. Please see #860

Comment (AlexKoff88) [reply docs/compression_algorithms/Quantization.md:308]: @asenina, can you explain what do you mean by a different behavior?

**Comment by asenina** [comment]
Jenkins please retry a build

**Review by alexsu52 — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]

**Review by vshampor — APPROVED** [review]

**Pull request merged**